manifest.yaml

```diff
@@ -2,13 +2,6 @@ name: github.com/getoutreach/stencil-base
 ## <<Stencil::Block(keys)>>
 modules:
   - name: github.com/getoutreach/devbase
```

Comment:
Should we say this requires `>=` the latest devbase?

Reply:
Good call.